easybuild/easyconfigs/s/SuperLU/SuperLU-5.2.1-foss-2017b.eb

```diff
@@ -0,0 +1,17 @@
+name = 'SuperLU'
+version = '5.2.1'
+
+homepage = 'http://crd-legacy.lbl.gov/~xiaoye/SuperLU/'
+description = """SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems
+ of linear equations on high performance machines."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'opt': True, 'pic': True, 'openmp': True}
```

Comment:
@hajgato `opt` is enabled by default, so please drop that one here

Reply:
`'opt': True` is `-O3` and `' opt': False` is `-O2`, or am I wrong?

Reply:
Ugh, yes, sorry, you're right.

So, you really want to build this with `-O3` rather than `-O2`?

Reply:
I just copied from the old versions.

Reply:
ok, then. `-O2`